### PR TITLE
chore: updating dependabot to support gha, python, docker and dev container packages

### PR DESCRIPTION
**Pull request by rajpalc7**

### CODE REVIEW

**Comment by rajpalc7** [comment]
Hi @WadeBarnes , this is ready for review now.

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  # Maintain dependencies for GitHub Actions
+  #  - Check for updates once a week
+  #  - Group all updates into a single PR
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    groups:
+      all-actions:
+        patterns: [ "*" ]
```
Comment (WadeBarnes) [inline .github/dependabot.yml]: There are no GitHub actions in this repo.

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  - package-ecosystem: "pip"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+      day: "monday"
+      time: "04:00"
+      timezone: "Canada/Pacific"
+    ignore:
+      - dependency-name: "*"
+        update-types: ["version-update:semver-major"]
```
Comment (WadeBarnes) [inline .github/dependabot.yml]: There is no python code in this repository.

File: .github/dependabot.yml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+      timezone: "Canada/Pacific"
+
+  # Maintain dependencies for dev Container
+  - package-ecosystem: "devcontainers"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+      day: "monday"
+      time: "04:00"
+      timezone: "Canada/Pacific"
```
Comment (WadeBarnes) [inline .github/dependabot.yml]: This is redundant

**Review by WadeBarnes — CHANGES_REQUESTED** [review]
@rajpalc7, Please have a closer look at the code.

File: .github/dependabot.yml (hunk 3 above)
Comment (rajpalc7) [reply .github/dependabot.yml]: Sorry @WadeBarnes there was confusion between orgbook-bc-api-docs and orgbook-bc-client. This should be fixed now.

**Review by WadeBarnes — CHANGES_REQUESTED** [review]
@rajpalc7, Please have a closer look at the code.  You are missing an update section for dependency manifests that do exist in the repo.

**Comment by rajpalc7** [comment]
@WadeBarnes Fixed

**Review by WadeBarnes — APPROVED** [review]

**Pull request merged**